main.js

```diff
@@ -0,0 +1,69 @@
+let bookCollection = [];
+
+const removeBook = (id) => {
+  bookCollection = bookCollection.filter((book) => book.id !== id);
+
+  localStorage.setItem('bookCollection', JSON.stringify(bookCollection));
+};
+
+const displayBooks = () => {
+  const bookList = document.getElementById('book-list');
+  bookList.innerHTML = '';
+  bookCollection.forEach((book) => {
+    const bookContainer = document.createElement('article');
+    bookList.appendChild(bookContainer);
+    bookContainer.innerHTML = ''; // Clear the container
+
+    const titleElement = document.createElement('p');
+    titleElement.textContent = book.title;
+
+    const authorElement = document.createElement('p');
+    authorElement.textContent = book.author;
+
+    const buttonElement = document.createElement('button');
+    buttonElement.textContent = 'Remove';
+    buttonElement.setAttribute('data-id', book.id);
+    buttonElement.addEventListener('click', () => removeBook(book.id));
+
+    const hrElement = document.createElement('hr');
+
+    bookContainer.appendChild(titleElement);
+    bookContainer.appendChild(authorElement);
+    bookContainer.appendChild(buttonElement);
+    bookContainer.appendChild(hrElement);
+  });
+};
+
+const addBook = () => {
```

Comment:
Also when I add book without author it adds it so include validation or requirements to add book with both title and author.